src/storage/mvcc/reader/reader.rs

```diff
@@ -594,7 +594,10 @@ impl<S: EngineSnapshot> MvccReader<S> {
         Ok((locks, has_remain))
     }
 
-    /// Scan the writes to get all the latest user keys. The return type is:
+    /// Scan the writes to get all the latest user keys. This scan will skip
+    /// `WriteType::Lock` and `WriteType::Rollback`, only return the key that
+    /// has a latest `WriteType::Put` or `WriteType::Delete` record. The return
+    /// type is:
```

Comment:
need to update as well

Reply:
I think this part covers all types already?

Reply:
Well, it makes sense for now :)